spec.emu

```diff
@@ -67,7 +67,30 @@ contributors: Ujjwal Sharma, Younies Mahmoud
       </table>
     </emu-table>
 
-    <emu-note type=editor>The [[Weeks]] field is still under discussion in Temporal. For more, check out the <a href="https://github.com/tc39/proposal-temporal/issues/532">Github thread</a> on the Temporal issue tracker.</emu-note>
+    <emu-clause id="sec-getunitoptions" aoid="GetUnitOptions">
+      <h1>GetUnitOptions ( _unit_, _options_, _baseStyle_, _timeUnit_ )</h1>
+      <p>The GetUnitOptions abstract operation is called with the arguments _unit_ (which must be a String), _options_ (which must be an Object) and _baseStyle_ (which must be a String) and returns a Record containing the relevant options for that unit. The following steps are taken:</p>
+
+      <emu-alg>
+        1. Let _stylesList_ be the List &laquo; `"long"`, `"short"`, `"narrow"` &raquo;.
```

Comment:
What happens with weird styles, e.g. `{hours: numeric, minutes: 'long'}`.  Error?  Does "long" get upgraded to 2-digit?  Something else?